public/static/locales/zh-CN/resourceLibrary.json

```
@@ -3,29 +3,28 @@
   "sider-title2": "用户指南",
   "sider-title3": "学院（敬请期待）",
   "sider-title4": "博客",
-  "sider-title5": "开始入门（敬请期待）",
+  "sider-title5": "开始入门",
   "sider-title6": "教程",
   "sider-title7": "视频库",
   "sider-content-title1": "开发者文档",
   "sider-content-title1-list1": "开始",
-  "sider-content-title1-list2": "安装部署",
-  "sider-content-title1-list3": "开发指南",
-  "sider-content-title1-list4": "更新和迁移",
-  "sider-content-title1-list5": "开发者资源",
-  "sider-content-title1-list6": "用户指南",
+  "sider-content-title1-list2": "安装依赖",
+  "sider-content-title1-list3": "安装指南",
+  "sider-content-title1-list4": "Admin API",
+  "sider-content-title1-list5": "Control API",
   "sider-content-title2": "用户指南",
-  "sider-content-title2-list1": "开始",
-  "sider-content-title2-list2": "内容管理者",
-  "sider-content-title2-list3": "内容类型生成器",
-  "sider-content-title2-list4": "用户、角色和权限",
-  "sider-content-title2-list5": "插件",
+  "sider-content-title2-list1": "快速入门",
+  "sider-content-title2-list2": "健康检查",
+  "sider-content-title2-list3": "开发插件",
+  "sider-content-title2-list4": "使用多语言开发插件",
+  "sider-content-title2-list5": "常见问题",
   "sider-content-title2-list6": "通用设置",
   "sider-content-title3": "资源",
   "sider-content-title3-list1": "贡献者趋势",
   "sider-content-title3-list2": "支持",
   "sider-right-center-title": "资源中心",
   "sider-right-center-title1": "资源库",
-  "sider-right-center-desc1": "更好地使用 Apache APISIX 所需的一切。",
+  "sider-right-center-desc1": "优化你使用 Apache APISIX的工作方式所需要的一切",
```

```suggestion
  "sider-right-center-desc1": "优化你使用 Apache APISIX 的工作方式所需要的一切",
```

🤔 @yzeng25 any other descriptions here?